flight/Libraries/sanitycheck.c

```diff
@@ -264,6 +264,13 @@ static int32_t check_safe_to_arm()
 			case FLIGHTSTATUS_FLIGHTMODE_STABILIZED3:
 			case FLIGHTSTATUS_FLIGHTMODE_ALTITUDEHOLD:
 				break;
+
+			case FLIGHTSTATUS_FLIGHTMODE_FAILSAFE:
```

Comment:
should note in the doxygen docs for the function (or make some), or rename it. As is there it is implied that it's safe to arm while in failsafe mode.